README.rst

```diff
@@ -160,8 +160,29 @@ The geographic app is based on `django-loci <https://github.com/openwisp/django-
 and allows to define the geographic coordinates of the devices,
 as well as their indoor coordinates on floorplan images.
 
-This module also provides an API through which mobile devices can update
-their coordinates.
+This module also supports the following API endpoint.
+
+* ``/api/v1/device/{id}/location/``
+
+This API endpoint supports the ``GET`` and ``PUT`` method which is used by
+mobile devices to get and update their location coordinates.
+
+Example usage of endpoint with ``GET`` method
+
+.. code-block:: shell
+
+    http GET localhost:8000/api/v1/device/b328ccec-50e4-4fef-bf00-be726bd632e6/location/?key=abe3c18b5c6c253b9e53c1d69797353a
+
+Example usage of endpoint with ``PUT`` method
+
+.. code-block:: shell
+
+    import json
+    from django.contrib.gis.geos import Point
+
+    coords = json.loads(Point(2, 23).geojson)
+    DATA = {'type': 'Feature', 'geometry': coords}
```

Comment:
this example mixes python and shell and is totally broken, please avoid mixing and make sure the example runs fine in the shell when copying it and pasting it.

Instead of using python to build the geojson string, just provide it ready:

`DATA = '{"type": "Feature", .....'`